Getting-Started/linux.md

```diff
@@ -1,81 +1,68 @@
 
-As the name suggests, `cd` stands for "Change Directory" (this will be a large topic, encompassing file paths, . and .. ) Try changing directories to your Desktop directory by typing `cd Desktop`. Keep in mind the shell is case sensitive, so be sure to replicate capitalization.
+On your Linux machine, go to the [VSCode website](https://code.visualstudio.com/download) and download the `.deb` file. Change directories to your Downloads (`cd Downloads`) and use the `dpkg` command with the `-i` suffix to install VSCode (`dpkg -i PACKAGE_NAME`). To launch VSCode, simply switch to a directory with a project and type `code .` which will launch VSCode in the current directory.
 
-#### But wait, I don't have to type the entire name?
+_Now that they have it installed, do we want to walk them through extensions?_
 
-No you don't. Type `cd ..` and then `cd Dow[and then press tab]`. This should complete your entry to `cd Downloads/`. Hit enter and you should be in your Downloads folder. Pretty powerful, right?
+### Anaconda
 
-#### ls
+Now that we've got a coding environment, let's install Anaconda. Wright State's native computer wizard Ryan Slater explains Anaconda pretty well:
 
-So we've covered how to see where you're at in the file system, how do you see what's inside the folder you're currently in? Well, theres a command for that. Type `ls` and hit enter. The contents of whatever directory you're in (check it with `pwd`!) should be printed out to the screen. But, that's just the name of the files and folders, which is LAME. To print out all the cool nerdy stats, type `ls -l` (standing for ls long). This should print out a bunch of cool information about the files and folders that are in your current working directory. To see hidden files, type `ls -al`. Well, now you know what's inside a directory - how do we make files to populate it?
+> The big reason to use Anaconda over plain old boring Python is how Anaconda handles environments. Environments are basically virtual machines, but for Python. They have some version of Python installed, and some specified set of modules. They are completely separate and independent from your core Python installation, meaning you can easily spin up different combinations of modules and versions without affecting your host OS. This is great for testing your code on old versions of modules, trying beta features, or for setting up a module that might break some other module you use. Below are a list of useful commands. Check out the [documentation](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands) for more information.
 
-Arch stinks LOL
+If you want more of that, check out Ryan's Python guide [here](https://github.com/cse-devteam/Programmers-Guide-to-the-Galaxy/blob/master/Getting-Started/python.md). Really, you should.
 
-SSH Keys are awesome!
+What else should we add here?
```

Comment:
Directory/file permissions?
Might explain the "branches" of Linux distros, i.e. Debian, Fedora, etc
![image](https://user-images.githubusercontent.com/20517031/119388629-88bbec00-bc98-11eb-880d-5e179c67b2f7.png)

Reply:
I inserted stuff about file and directory permissions, but decided to omit stuff about the linux distro tree, I don't think it will be useful in the long run.